emissionscheck_alb/data.py

```diff
@@ -15,36 +15,59 @@ def keep_pass_fail(df):
         DataFrame of rows with desired values `'P'` or `'F'` in target variable 
     '''
     # Mask to select only rows with desired target values
-    overall_result_mask = (df['OVERALL_RESULT'] == 'P') | (df['OVERALL_RESULT'] == 'F')
+    overall_result_mask = (data['OVERALL_RESULT'] == 'P') | (data['OVERALL_RESULT'] == 'F')
     
-    return df[overall_result_mask]
+    return data[overall_result_mask]
 
-def get_car_age(df):
+def get_car_age(data):
     '''
     Calculates age of car based on model year and year of emissions test.
-    
+
     Parameters
     ----------
     df : pandas.DataFrame
         DataFrame containing the columns `TEST_EDATE` and `'MODEL_YEAR'`
         `'MODEL_YEAR'` should be `int` type
-    
+
     Returns
     -------
     pandas.DataFrame
         DataFrame of rows with new column `CAR_AGE`
-        
-    
+
+
     Notes
     -----
     Adding +1 to car age to avoid negative values (many cars are released a year prior to year in car model)
-    
+
     '''
+    # make copy of df
+    df_copy = data.copy()
+
     # convert df['TEST_EDATE'] to datetime
-    df_copy = df.copy()
+
     df_copy.loc[:, 'CAR_AGE'] = pd.to_datetime(df_copy.loc[: ,'TEST_EDATE']).dt.year - df_copy.loc[:, 'MODEL_YEAR']
-    
+
     # add +1 to df['CAR_AGE']
     df_copy.loc[:, 'CAR_AGE'] += 1
-    
-    return df_copy
\ No newline at end of file
+
+    return df_copy
+
+def encode_target(target_series):
+    '''
+    Encode OVERALL_RESULT `"P"` or `"F"` in target variable `OVERALL_RESULT`
+
+    Parameters
+    ----------
+    df : pandas.DataFrame

```

Comment:
are you importing a dataframe or a series? docstring says df, but code suggests series

Reply:
The function accepts a DataFrame, makes a copy of it, and then takes the relevant columns.

Reply:
hmm, am I looking at an old version again? in here, the function takes a series, modifies it and returns it. the comment says otherwise
![image](https://user-images.githubusercontent.com/34165505/84576666-d5275800-adb6-11ea-9134-b220266b766f.png)